test_is_appropriate.py

```diff
@@ -0,0 +1,9 @@
+"""Verify if the function is_appropriate(number) returns True when an
+appropriate number is given."""
+from FizzBuzz import is_appropriate
+
+def test_is_appropriate():
+    """Assert that integers between 0 and 101 return True."""
+    assert is_appropriate(1) is True
```

Comment:
after you refactor is_appropriate to be a predicate rather than assertion, some of these tests should return false